readme-developer.md

```diff
@@ -1,158 +1,121 @@
-# Build instructions
+# Setup
 
-Most of the current development work is done on Ubuntu Linux. Building the code on any Linux variant should be relatively straightforward. Building it on Windows or Mac OS X is a bit more complicated.
+First you need a copy of the code:
 
-**You can get a copy of the code either using "git clone," or using the repo's download button to obtain a [.ZIP archive](https://github.com/endless-sky/endless-sky/archive/master.zip).** The game's root directory, where your unzipped/`git clone`d files reside, will be your starting point for compiling the game.
+```powershell
+> git clone https://github.com/endless-sky/endless-sky --recursive
+```
+
+If you clone the repo another way make sure to download submodules as well when doing so. If you didn't you can download the submodule by executing
+
+```powershell
+> git submodule update --init
+```
 
-How you build Endless Sky will then depend on your operating system:
+The game's root directory, where your `git clone`d files reside, will be your starting point for compiling the game.
 
+How you build Endless Sky will then depend on your IDE and your operating system.
 
 
 ## Linux
 
 Use your favorite package manager to install the following (version numbers may vary depending on your distribution):
 
-##### DEB-based distros:
-   - g++
-   - scons
-   - libsdl2-dev
-   - libpng-dev
-   - libjpeg-dev
-   - libgl1-mesa-dev
-   - libglew-dev
-   - libopenal-dev
-   - libmad0-dev
-   - uuid-dev
-
-##### RPM-based distros:
-   - gcc-c++
-   - scons
-   - SDL2-devel
-   - libpng-devel
-   - libjpeg-turbo-devel
-   - mesa-libGL-devel
-   - glew-devel
-   - openal-soft-devel
-   - libmad-devel
-   - libuuid-devel
-
-Then, from the project root folder, simply type:
+<details>
+<summary>DEB-based distros</summary>
 
 ```
-    $ scons
-    $ ./endless-sky
+g++ cmake ninja-build pkg-info libgl1-mesa-dev libxmu-dev libxi-dev libglu1-mesa-dev
 ```
 
-The program will run, using the "data" and "images" folders that are found in the project root. For more Linux help, and to view other command-line flags, consult the `man` page (endless-sky.6), by running `man endless-sky` in the project directory.
-The program also accepts a `--help` command line flag.
+If you're on an older version of Ubunut, you'll need an up-to-date version of cmake which you can get from the [offical apt](https://apt.kitware.com/).
```

Comment:
kitware does look quite official, but isn't cmake.org/download a more appropriate download link?

Reply:
It's official, it's even linked on the download page you mentioned (towards the end, under "Alternative Binary Releases").

I'm still in favour of using the cmake download page, because it also links to other methods of distribution (snap, pip).

Reply:
To be clear, this specific distribution of MinGW is used by chocolatey, which in turn is used by GitHub and used in their official Windows images. So I'd say it's trustworthy?